plutus-tx-plugin/src/PlutusTx/PIRTypes.hs

```diff
@@ -1,13 +1,14 @@
 module PlutusTx.PIRTypes where
 
 import PlutusIR qualified as PIR
+import PlutusTx.Annotation
 
-type PIRKind = PIR.Kind ()
-type PIRType uni = PIR.Type PIR.TyName uni ()
-type PIRTerm uni fun = PIR.Term PIR.TyName PIR.Name uni fun ()
+type PIRKind = PIR.Kind Ann
+type PIRType uni = PIR.Type PIR.TyName uni Ann
+type PIRTerm uni fun = PIR.Term PIR.TyName PIR.Name uni fun Ann
 type PIRProgram uni fun = PIR.Program PIR.TyName PIR.Name uni fun ()
```

Comment:
Currently there's no need to have annotation on PIRProgram/PLCProgram/UPLCProgram, since once have a Program, the compilation is done and the only thing we do to it is `dumpFlat`.

Should we keep it as it is until we need to annotation the Program?